exporter/containerimage/writer.go

```diff
@@ -111,10 +112,15 @@ func (ic *ImageWriter) Commit(ctx context.Context, inp *exporter.Source, session
 			ref = inp.Ref
 		} else if len(p.Platforms) > 0 {
 			p := p.Platforms[0]
-			if _, ok := inp.Attestations[p.ID]; ok {
-				return nil, errors.Errorf("cannot export attestations without multi-platform enabled")
-			}
 			ref = inp.Refs[p.ID]
+			if atts, ok := inp.Attestations[p.ID]; ok {
+				atts = attestation.Filter(atts, nil, map[string][]byte{
+					result.AttestationInlineOnlyKey: []byte(strconv.FormatBool(true)),
+				})
+				if len(atts) > 0 {
+					return nil, errors.Errorf("cannot export attestations without multi-platform enabled")
```

Comment:
When does this error appear? Doesn't `ForceIndex` always make result multiplatform when attestations are present?